### ISSUE TITLE
2022-1-29

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#725]
481	What is a state_dict? In PyTorch, the learnable parameters (i.e. weights and biases) of an torch.nn.Module model are contained in the model’s parameters (accessed with model.parameters()). A state_dict is simply a Python dictionary object that maps each layer to its parameter tensor. only layers with learnable parameters (convolutional layers, linear layers, etc.) and registered buffers (batchnorm’s running_mean) have entries in the model’s state_dict. Optimizer objects (torch.optim) also have a state_dict, which contains information about the optimizer’s state, as well as the hyperparameters used. Only appear in save and load; Because state_dict objects are Python dictionaries, they can be easily saved, updated, altered, and restored, adding a great deal of modularity to PyTorch models and optimizers. (2022-1-21)
482	'early_stop': 200,               # early stopping epochs (the number epochs since your model's last improvement) ; (2022-1-21)
483	Test不是应该给出一个准确率或者loss吗? Why it generates a file? (2022-1-21)
484	HW4, wtf? Dataset is 7G? (2022-1-21)
485	Go over pytorch tutorial; The essential problem is that we do not know the coding format of pytorch, so going over pytorch tutorial and modifying it might be the best way to be masterful of it; 
486	The IPython Notebook is now known as the Jupyter Notebook; (2022-1-21)
487	Have hair cut; done; (2022-1-22)
488	What is grad_fn? grad_fn is a function "handle", giving access to the applicable gradient function. The gradient at the given point is a coefficient for adjusting weights during back-propagation. (2022-1-23)
489	Check requirement and format of thesis; download related files; done; (2022-1-23)
